### DISCUSSION TITLE
Is Overseer still an active project?

### DISCUSSION THREAD

**Discussion by mikekenn** [sct/overseerr#3991] [Q&A]
My version (1.33.2) is showing it's been 500 days since its last upgrade. It's being monitored by Watchtower and I can confirm it is checking daily to see if there are updates?

Im just wondering if 500+ days is the normal release cycle for this project or if everything has just gone stale?

Thanks.

**Comment by john-lazarus** [comment]
Been wondering the same..

**Comment by Lockszmith-GH** [comment]
See #3948.
It's active, There has been a lot of work done - that's for sure.
But it's been a while since a release was created.

**Reply by ChipSkylark37** [reply]
If there have been a lot of commits, is there any way to get this into our current release? Or do we need to wait until a new release?

**Reply by Lockszmith-GH** [reply]
I wasn't updating for a long time either.

You make a good point. I do see the owner, hasn't been very active on this repo for a long time - but it is clear some contributors have merge-privileges.
Theoretically, anyone can spin a docker-build based on the Dockerfile and build an image.

Have you tried [linuxserver/overseerr:develop](https://hub.docker.com/r/linuxserver/overseerr)?
It looks like their automation keeps on churning, and they are a solid group.

**Reply by ChipSkylark37** [reply]
Does that branch have new updates to the overseer application itself, or just the base OS image it runs on? It says 'application updates' but if its just basing that off the release version from SCT then it will be the same 'old' version.

**Reply by Lockszmith-GH** [reply]
That's why I suggested them, they clearly state in the Versions Tag section:

> |  Tag | Available | Description |
> |  -- | -- | -- |
> |  latest | ✅ | Stable releases from GitHub |
> |  develop | ✅ | Development releases from commits in upstream develop branch |

The upstream repo is sct/overseerr (this repo), so that means the develop branch here, with a last commit 2 month ago.

BTW, the [sctx/overseerr:develop tag](https://hub.docker.com/layers/sctx/overseerr/develop/images/sha256-483969de63c744df5ed33bd99eb08a20d9aaa12c24ba8babdd7de155b9d165b2) was updated 2 month ago - so it seems automation actually is working.

**Reply by ChipSkylark37** [reply]
Thanks - I switched over to the linuxserver develop branch.
